File: API/Core/Entities/JobItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace API.Core.Entities
{
    public class JobItem
    {
        public int Id { get; set; }
        public int CustomerId { get; set; }
        public string Title { get; set; }
        public string JobDescription { get; set; }
        public DateTime JobDate { get; set; }
        public decimal Amount { get; set; } = 0M;
        public bool IsInvoiced { get; set; }
        public Customer Customer { get; set; }

    }
}

File: client/src/app/shared/Models/JobItem.ts
export interface IJobItem {
    id: number;
    customerId: number;
    title: string;
    jobDescription: string;
    jobDate: Date;
    amount: number;
    isInvoiced: boolean;
}

export class JobItem {
    id: number;
    customerId: number;
    title: string;
    jobDescription: string;
    jobDate: Date;
    amount: number;
    isInvoiced: boolean;
}
